packages/core/chrome/core-chrome-browser-internal/src/ui/project/app_menu.tsx

```diff
@@ -28,7 +29,7 @@ export const AppMenuBar = ({ headerActionMenuMounter }: AppMenuBarProps) => {
         display: flex;
         justify-content: end;
         align-items: center;
-        padding: ${euiTheme.size.s};
+        height: var(--kbnProjectHeaderAppActionMenuHeight, ${euiTheme.size.xxxl});
```

Comment:
Making the app menu height fixed. 

It is also important to not render the menu if there are no actions. There is no good way for app menu component to know if there is content inside. We'd try to improve places where apps set the menu mount point, so that apps stopped setting the mount point if there is no actions

Reply:
This feels super elegant - it's also interesting that a plugin could theoretically update the `--kbnProjectHeaderAppActionMenuHeight` CSS variable and have everything update automatically, if they don't want to use the default `xxxl` height. Feels really flexible / future-proof!